### ISSUE TITLE
Define "Information" and "Synonym"

### ISSUE THREAD

**Issue by gryphonshafer** [gryphonshafer/Quizzing-Rule-Book#66]
We use the word "information" a lot in the rule book, but we don't define it. We need a good definition of "information", which cannot be the same as "material", because that would mean any synonym would not be acceptable to be correct.

We also never talk about the concepts behind the word "synonym". And we don't use "synonym" in any rule. And yet probably all QMs (certainly nearly all) conceptually consider the concept. So we should add that to the rule book and make it explicit/objective.

**Comment by scottpeterson** [comment]
I wouldn't want to use the word synonym in the rule book. What QMs are considering is if the answer of the quizzer meets requirements (of being correct, of being incorrect, of being out of context). So "saying a word that means basically the same thing" may be ruled as enough....and I don't want to potentially over-simplify rulings by letting people lean back on the word synonym.

**Comment by gryphonshafer** [comment]
I'm OK using the word "synonym" so long as we explicitly define it (with a term/definition in some reasonable section). In that way, we can prevent the over-tight interpretation of the word. But sans that, I agree we should avoid the use of the word.

**Comment by jswingle** [comment]
Agreed on "information." That's a technical term used throughout the rulebook.

As for "synonym" I only find it in the full-content render one time, under "3. Quiz Process" where it describes "response" as "a synonym for answer." This is a straightforward English usage of "synonym" and we don't need a technical definition for just this one usage.

**Comment by jswingle** [comment]
@gryphonshafer Would you like to come up with a definition for "information" that we can put in the definitions section of the rulebook?

**Comment by gryphonshafer** [comment]
> **Information** : Material content consisting of the question text and answer text including word or phrase variance thereof that maintains synonymous meaning

**Comment by jswingle** [comment]
Some things to consider:

"Incorrect information" is a phrase used a few times, and "wrong information" appears exactly once (an inconsistency which ought to be corrected).

"Information" is also used in a few contexts which is not applicable to the definition you just gave, which means we either need to adjust the definition or change those contexts.

Should we define "incorrect information" as a unique term separate from "information"?

**Comment by gryphonshafer** [comment]
I think "incorrect information" and "wrong information" (and yes, agreed, we should pick one or the other) just mean "stuff that's beyond the limits of 'information'".

> "Information" is also used in a few contexts which is not applicable to the definition you just gave, which means we either need to adjust the definition or change those contexts.

Can you cite these?

**Comment by jswingle** [comment]
Ctrl+F on the full render and several of the first few hits are incompatible

**Comment by gryphonshafer** [comment]
Which ones?

Here's what search returns: https://github.com/gryphonshafer/Quizzing-Rule-Book/search?q=information

**Comment by jswingle** [comment]
"No comment other than "correct" or "incorrect" need be announced by the quiz officials when announcing a ruling; however, at the discretion of the quizmaster (QM) and answer judge (AJ), or at the request of the captain, additional information may be given.

If the additional information regarding the ruling is provided at the discretion of the QM/AJ and not at the request of the captain, this is not considered an explanation of a ruling, which would invalidate a challenge or protest.

If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling and invalidates a challenge or protest. Before replying to a request for an explanation, the QM must ask all teams if they wish to challenge."

**Comment by gryphonshafer** [comment]
Sorry, maybe I'm being dense, but how does that contradict the proposed definition?

**Comment by gryphonshafer** [comment]
How about we change "information" in the above to "enlightenment"? ;)

Just kidding. Honestly, I think "additional information" above works OK with the proposed definition except for "additional information regarding the ruling", which maybe we could change to just "explanation"?

**Comment by jswingle** [comment]
Sure, as long as it doesn't use a technically defined term in a way that makes no sense =)

**Comment by jswingle** [comment]
Language proposal change for terminological consistency, which requires changing FOUR different sections of the rulebook. I think this is all much better, but I hope the part that will generate debate is the two new definitions I'm offering.

--

4.4.1 Example

**Original:**
"As an example of a logistical rule, consider a scenario wherein the quizmaster did not allow the full question answer duration period to expire for the quizzer when the quizzer was in context and had not given any wrong information."

**Proposal:**
"As an example of a logistical rule, consider a scenario wherein the quizmaster did not allow the full question answer duration period to expire for the quizzer when the quizzer was in context and had not given any _incorrect_ information."

--

2.2 Quiz Officials
**Original:**
"No comment other than "correct" or "incorrect" need be announced by the quiz officials when announcing a ruling; however, at the discretion of the quizmaster (QM) and answer judge (AJ), or at the request of the captain, additional information may be given.

If the additional information regarding the ruling is provided at the discretion of the QM/AJ and not at the request of the captain, this is not considered an explanation of a ruling, which would invalidate a challenge or protest.

If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling and invalidates a challenge or protest. Before replying to a request for an explanation, the QM must ask all teams if they wish to challenge."

**Proposal:** (compare with Section 4.3, I think the two sections work together nicely with this revision)

"No comment other than "correct" or "incorrect" need be announced by the quiz officials when announcing a ruling; however, at the discretion of the quizmaster (QM) and answer judge (AJ), or at the request of the captain, additional _explanation_ may be given.

If the additional _explanation_ regarding the ruling is provided at the discretion of the QM/AJ and not at the request of the captain, this _does not_ invalidate a challenge or protest.

If the captain requests additional _explanation_ regarding the ruling, _this invalidates a challenge or protest_. Before replying to a request for an explanation, the QM must ask all teams if they wish to challenge."

--

In 3. Quiz Process, we add two new definitions (different from Gryphon's definition):

Information: "The meaning of the material, including any non-verbatim words said by a quizzer during the answering period which maintain synonymous meaning with the material"

Incorrect Information: "Words said by a quizzer during the answering period which are substantially different in meaning from the information in the material"

MY EXPLANATION: We want to maintain the category of a quizzer saying things that are not yet "information" but also not so substantially different from the material that they are considered "incorrect information." The rulebook says that a quizzer is automatically incorrect once they have "given incorrect information," so we don't want to make it such that a quizzer can ONLY say things that are either "information" or "incorrect information." There has to be a gray area that is simply _neither_.

We also do not want to consider ONLY the question and answer text in the term "information," as Gryphon proposed, because a quizzer is allowed to quote from the context so long as their quoting is reasonably accurate and not "incorrect information." The meaning of the entire material should be considered "information," though some "information" is also "out of context" information.

--

In 1. Material and Questions, we have a list of definitions, but the word "material" is defined outside the list. This should be fixed.

Original:
"Material refers to the Scriptural scope of content set for each season."

Proposal: Included in the definitions list in section 1:
"Material: The Scriptural scope of content set for each season."

**Comment by kclimenhaga** [comment]
I agree with Jeremy's proposed changes to 4.4.1, 2.2, and 1. For the Quiz Process, I'm not convinced we need to add these definitions at all, but if we do add definitions, I would like to see them written at a more quizzer-friendly reading level. 

According to Gryphon's search linked above, I believe the only uses of "information" we would be left with are:

_Giving the name of a deity that is not correct is considered giving incorrect information._

(an answer is correct if) _- It contains the information requested_

To me, both of these are clear without further definitions. If the first one isn't clear, then I would rather reword it to "Giving the name of a deity that is not correct _will make the quizzer incorrect_" (with whatever other changes we made for the Deity/Trinity rule). I don't think that any definition for "information" could possibly make the second statement clearer.

**Comment by gryphonshafer** [comment]
> Information: "The meaning of the material, including any non-verbatim words said by a quizzer during the answering period which maintain synonymous meaning with the material"

> Incorrect Information: "Words said by a quizzer during the answering period which are substantially different in meaning from the information in the material"

I'm cool with the above 2 terms/definitions if we remove the word "substantially" from the second. It's subjective, and there's already a mutually-exclusive difference between "synonymous meaning" and "different in meaning".

**Comment by jswingle** [comment]
Agreed with Gryphon's revision. Also, in the first proposal I made above, note that the term "question answer duration period" has already been changed to "answering period" in the upcoming rulebook revision, so my copy+paste is a little wrong. It doesn't affect the substance of the change at all.

**Comment by jswingle** [comment]
@kclimenhaga I'd definitely like these definition proposals to be picked apart as much as possible, and readability is key.

@gryphonshafer I think the word "substantially" is important here, and I don't think it's an issue for it to be subjective. We don't want the definition to just say "different in meaning," because that would make it way too easy for a quizzer to "give incorrect information." The word "substantially" indicates to officials that they're not supposed to be nitpicking tiny nuanced differences.

**Comment by ARMediting** [comment]
2.2.1.1.3. also is a different use of "information." We could change it out for "some indication."

-----

Also, @jswingle I want to make sure I'm understanding this correctly:

> Incorrect Information: "Words said by a quizzer during the answering period which are substantially different in meaning from the information in the material"

If "material" encompasses the entire season's verses, then the definition of "incorrect information" doesn't cover accurately quoted words that are out of context of the question being asked, correct? (Also, based on Jeremy's proposed definition of "Information," then "in the material" is redundant.)

-----

I agree with @kclimenhaga about making the definition more kid-friendly. 

**Proposal:** 
"Information: The subset of material a quizzer provides during the answering period, if not word-perfect, then deemed to be adequately similar by the quizmaster and answer judge."

"Adequately" could be changed to "substantially," but I think "adequately" gives more power to the officials.

-----

However, 2.2.1.1.3. also says, "once all of the information in both the question and answer has been given, but something needs to be fixed...". I think this contradicts all of our proposed definitions of "Information" and may just need reworded. (I'm also happy to just not define "Information" or "Incorrect Information.)

**Comment by jswingle** [comment]
That's a great catch @ARMediting . We could change it to "indication," but actually I would rather just get rid of that entire last part of the phrase: "as this would give the quizzer information about where their mistake was made." I feel like this is not only obvious but also not rulebook-appropriate language. We don't need to justify the rules within the rulebook, if someone wants a justification they should ask whoever wrote the rules. (Part of the reason we have this GitHub, lol.)

**Comment by jswingle** [comment]
@ARMediting is also right about the contradiction of terminology in 2.2.1.1.3.

We will need to either do a full-scale rewrite of 2.2.1.1.3, or refrain from defining "information" and "incorrect information" specifically.

Also, while it's an admirable effort, I think the other definition you provided is probably just as difficult for quizzers to understand. (I had to read it a couple times myself.) 

We may need to go back to the drawing board on this whole issue. I think there is an advantage to defining "information" and "incorrect information," but we may need to take a simpler approach, such as "The content required by the quizmaster's question," and remove the concept of "the things said by the quizzer" from the definition. This would require a rewording/reworking of some of the other uses of "information" in the rulebook beyond the ones we've looked at so far.
